projects/tech-for-better/build-1.md

```diff
@@ -5,7 +5,8 @@
 | Time    | Activity                     |
 | ------- | ---------------------------- |
 | 09:45   | Check-in                     |
-| 10:00   | Reach out to Project Mentors & Role mentors |
+| 10:00 | Feedback Survey |
+| 10:10   | Reach out to Project Mentors & Role mentors |
```

Comment:
I wasn't sure about this in SP build 2 so removed it - I'm not sure if we want to allocate a specific time or leave it to them to find each other as and when they need it?

Reply:
Happy to remove